src/builtins/BuiltinJSON.cpp

```diff
@@ -18,698 +18,23 @@
  */
 
 #include "Escargot.h"
-#include "runtime/GlobalObject.h"
 #include "runtime/Context.h"
-#include "runtime/VMInstance.h"
-#include "runtime/StringObject.h"
-#include "runtime/ArrayObject.h"
-#include "runtime/TypedArrayObject.h"
-#include "runtime/BooleanObject.h"
-#include "runtime/BigIntObject.h"
+#include "runtime/GlobalObject.h"
+#include "runtime/JSON.h"
 #include "runtime/NativeFunctionObject.h"
-
-#define RAPIDJSON_PARSE_DEFAULT_FLAGS kParseFullPrecisionFlag
-#define RAPIDJSON_ERROR_CHARTYPE char
-#include <rapidjson/document.h>
-#include <rapidjson/filereadstream.h>
-#include <rapidjson/memorystream.h>
-#include <rapidjson/internal/dtoa.h>
-#include <rapidjson/stringbuffer.h>
-#include <rapidjson/writer.h>
-#include <rapidjson/error/en.h>
+#include "runtime/VMInstance.h"
+#include "runtime/Value.h"
```

Comment:
This `Value.h` can be removed because `JSON.h` already includes this header file.